Example/Tests/GINICameraViewControllerTests.swift

```diff
@@ -34,5 +35,26 @@ class CameraViewControllerTests: XCTestCase {
         XCTAssertFalse(vc.captureButton.isEnabled, "capture button should be disaled when tooltip is shown")
         
     }
+    
+    func testCaptureImage() {
+        let expect = expectation(description: "image is captured")
+        vc = CameraViewController(successBlock: { document in
+            XCTAssertTrue(document.type == .image, "document should be an image")
+            expect.fulfill()
+        }, failureBlock: { _ in
+            XCTFail()
+            expect.fulfill()
+        })
+        
+        _ = vc.view
+        vc.viewWillAppear(true)
+        vc.captureButton.sendActions(for: .touchUpInside)
+        
+        waitForExpectations(timeout: 5.0) { error in
```

Comment:
Why so long? Should it be enough to schedule it during the next runloop frame? What I'm getting at, is that if it fails, it better fail immediately. Tests should be fast.

Reply:
Yes, you are right. It should take less than 1sec. 👍